src/sdk/map_api_types.js

```diff
@@ -445,29 +447,29 @@ class PartAPI {
             seq: this.seq,
             start: this.start,
             end: this.end,
-            uncommitted: true,
+            uncommitted: this._uncommitted,
```

Comment:
Lets make sure that this will not insert `uncommitted: false` keys to the parts in the DB because we don't want to store all these uneeded states and instead just omit this field from the part db.
Perhaps you can use `uncommitted: this._uncommitted ? true : undefined,` or with null, but not sure if this does the trick or not.

Reply:
ok

Reply:
Double checked. "uncommitted: false" is not stored in cached range part. So no need to change